drivers/SmartThings/matter-appliance/src/embedded-cluster-utils.lua

```diff
@@ -15,11 +15,10 @@ if version.api < 10 then
   clusters.TemperatureControl = require "TemperatureControl"
 end
 
-if version.api < 11 then
-  clusters.MicrowaveOvenControl = require "MicrowaveOvenControl"
-  clusters.MicrowaveOvenMode = require "MicrowaveOvenMode"
-  clusters.OvenMode = require "OvenMode"
-end
+--these clusters are not available in v10 and v11
+clusters.MicrowaveOvenControl = require "MicrowaveOvenControl"
+clusters.MicrowaveOvenMode = require "MicrowaveOvenMode"
```

Comment:
these should be gated by the version.api < 11 check. Only ovenMode can't be.

Reply:
Okay~!